HW1/P6/P6.txt

```diff
@@ -1 +1,12 @@
-# Your discussion here
+Ten randomly generated phrases:
+
+1.) indeed, that baes like a man of his own hand did make, Breathed forth the stocks! You stubborn ancient knave,
```

Comment:
Nice work. It looks like you are taking the most likely next word (instead of sampling from the count distribution) so some of the generated sentences have repeated phrases (e.g. Breathed forth the stocks!). However I think it's a fine solution.